tools/releaseTools.psm1

```diff
@@ -155,18 +171,22 @@ function Get-ChangeLog
         }
 
         # Find the commits that were made after the merge commit.
-        $new_commits_after_merge_commit = git --no-pager log --no-merges "$commit_hash..HEAD" --format=$format | New-CommitNode
+        $new_commits_after_merge_commit = @(git --no-pager log --first-parent "$commit_hash..HEAD" --format=$format | New-CommitNode)
```

Comment:
--first-parent find only the parent commit in a set of merge commits?

Reply:
confirmed... this explained it well for me: https://opendatascience.com/git-first-parent-have-your-messy-history-and-eat-it-too/

Reply:
Maybe the comment should be updated

Reply:
`--first-parent` basically means for any merge node, only follow the parent node on the master branch side. In case we merge a branch to master for a PR, only the merge node will show up in this way, the individual commits from that branch will be ignored. This is what we want because the merge commit itself already represents the PR.

I will update the comment to explain `--first-parent`